src/ibm_service_validator/cli/handlers/output_handler.py

```diff
@@ -160,26 +166,52 @@ def display_single_test(
 ) -> None:
     """Display an error or warning for a single method / endpoint."""
     default.display_subsection(result, color)
-    display_all_checks(unique_checks, color, result.seed)
+    display_all_checks(unique_checks, color, result.seed, result.path, result.method)
 
 
-def display_all_checks(checks: List[SerializedCheck], color: str, seed: int) -> None:
+def display_all_checks(
+    checks: List[SerializedCheck], color: str, seed: int, path: str, method: str
+) -> None:
     for idx, check in enumerate(checks, 1):
         message: Optional[str]
         if check.message:
             message = f"{idx}. {check.message}"
         else:
             message = None
         example = cast(SerializedCase, check.example)
-        display_example(example, color, check.name, message, seed)
+        display_example(example, color, method, path, check.name, message, seed)
         # Display every time except the last check
         if idx != len(checks):
             click.echo("\n")
 
 
+def produce_curl_command(
+    case: SerializedCase, path: str, method: str, request_example: str
```

Comment:
Reiterating what I said above. The `case` should have the `path` and `method` info.